pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,10 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Returns the filesystem path of the pod's network namespace; if the
+	// runtime does not handle namespace creation itself, or cannot return
+	// the network namespace path, it should return an error.
+	GetNetNS(containerID ContainerID) (string, error)
```

Comment:
TODO sounds good. We'll probably include this via the pod sandbox interface in the future, so container id is not needed.